layouts/partials/footer.html

```diff
@@ -33,7 +33,7 @@ <h3 class="footer-header pb-1">
       {{ end }}
     </div>
     <div class="py-5 footer-copyright">
```

Comment:
Maybe put a

```html
{{ with .Site.Params.copyrightMessage }}
<div class="py-5 footer-copyright">
<!-- the rest -->
{{ end }}
```

Reply:
will do!

Reply:
with broke the site. but If statement works :)

Reply:
Interesting, but quick fix if we get it to work with `with` later.